recipes/ipyurl/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "0.1.2" %}
+{% set sha256 = "9831f53ed6c564b59d72a047e627785139069e4bc0311d3393fd9fc6f52389bb" %}
+
+package:
+  name: ipyurl
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/i/ipyurl/ipyurl-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
```

Comment:
Please remove that in the feedstock again and do the mentioned changes to `conda-forge.yaml`.

Reply:
You mean, not here in the recipe but when the feedstock is published, right?